wikipedia-sentences/README.md

```diff
@@ -1,50 +1,76 @@
+
 # Semantic Wikipedia Search with Transformers and DistilBERT
 
 ![](https://docs.jina.ai/_images/jinabox-wikipedia.gif)
 
-| item   | content                                          |
-|--------|--------------------------------------------------|
-| Input  | 1 text file with 1 sentence per line             |
-| Output | *top_k* number of sentences that match input query |
 
-This is an example of using [Jina](http://www.jina.ai)'s neural search framework to search through a [selection of individual Wikipedia sentences](https://www.kaggle.com/mikeortman/wikipedia-sentences) downloaded from Kaggle. It uses the [`distilbert-based-uncased`](https://huggingface.co/distilbert-base-uncased) language model from [Transformers](https://huggingface.co).
+## Table of contents: 
+- [Overview](#overview)
+- [🐍 Build it yourself and deploy with Python](#---build-it-yourself-and-deploy-with-python)
+- [Overview of the files in this example](#overview-of-the-files-in-this-example)
+- [Flow diagram](#flow-diagram)
+- [Next steps, building your own app](#next-steps--building-your-own-app)
+- [🐳 Deploy the prebuild application using Docker](#---deploy-the-prebuild-application-using-docker)
+- [Community](#community)
+- [License](#license)
 
-## 🐳 Run in Docker
+# Overview
+|  |  |
+| ------------- | ------------- |
+| Summary | This showcases a semantic text search app |
+| Data for indexing | Wikipedia corpus |
+| Data for querying | A text sentence  |
+| Dataset used |  [Kaggle Wikipedia corpus](kaggle.com/mikeortman/wikipedia-sentences)     |
+| ML model used |  [`distilbert-based-uncased`](https://huggingface.co/distilbert-base-uncased) |
 
-To test this example you can run a Docker image with 30,000 pre-indexed sentences:
+This example shows you how to build a simple semantic search app powered by [Jina](http://www.jina.ai)'s neural search framework. You can index and search text sentences from Wikipedia using a state of art machine learning  [`distilbert-based-uncased`](https://huggingface.co/distilbert-base-uncased) language model from the [Transformers](https://huggingface.co) library.
 
-```sh
-docker run -p 45678:45678 jinahub/app.example.wikipedia-sentences-30k:0.2.10-1.0.10
-```
+# 🐍 Build the app with Python
+
+These instructions explain how to build the example yourself and deploy it with Python. If you want to skip the building steps and just run the app, check out the Docker section below.  
 
-You can then query by running:
+## Pre requirements:
+1. You have a working Python 3.7 or 3.8 environment. 
+2. We recommend creating a [new python virtual envoriment](https://docs.python.org/3/tutorial/venv.html) to have a clean install of Jina and prevent dependency conflicts.   
```

Comment:
- speling (environment)
- CaPitalizAtioN (Python)

Reply:
will make a pr for the template

Reply:
Thanks, corrected all cases of Python to uppercase (outside of code snippets) and fixed that spelling

Reply:
Thanks, fixed all cases of Python and spelling